nav2_costmap_2d/plugins/obstacle_layer.cpp

```diff
@@ -760,4 +768,22 @@ ObstacleLayer::resetBuffersLastUpdated()
   }
 }
 
+CombinationMethod combination_method_from_int (const int value, const std::string function_name){
```

Comment:
Remove function name, that's a pretty odd choice. Also, use the node's logger, don't create your own